modules/opening/src/main/OpeningSearch.scala

```diff
@@ -47,6 +47,7 @@ private object OpeningSearch:
         .take(200)
         .replace("_", " ")
         .replace("-", " ")
+        .replace(".", " ")
```

Comment:
what does this do? Tests say it's already handled

https://github.com/lichess-org/lila/blob/master/modules/opening/src/test/OpeningSearchTest.scala#L29-L30